import/esri_arcgisruntime_toolkit_qml.qrc

```diff
@@ -4,12 +4,16 @@
         <file>+cpp_api/CoordinateConversionOption.qml</file>
         <file>+cpp_api/CoordinateConversionResult.qml</file>
         <file>+cpp_api/NorthArrowController.qml</file>
+        <file>+cpp_api/PopupViewController.qml</file>
         <file>CoordinateConversion.qml</file>
```

Comment:
I wonder if it would be nice to split the QML and Cpp files in the qrc up into distinct sections

Reply:
What are you envisioning?